### PR TITLE
New data

### PR DESCRIPTION
**Pull request by MaxNoe**

### CODE REVIEW

File: intro/content/intro.tex [hunk 1]
```diff
@@ -84,28 +89,27 @@ \section{Ablauf}
...
       \end{itemize}
-    \item[Dienstag] Erstellen von Plots / Auswerten / Fehlerrechnung
+    \item[Mittwoch] Auswerten / Fehlerrechnung
       \begin{itemize}
-        \item matplotlib
         \item scipy
         \item uncertainties
       \end{itemize}
-    \item[Mittwoch] Kommandozeile und Automatisierung
+    \item[Donnerstag] Kommandozeile und Automatisierung
```
Comment (chrbeckm) [inline intro/content/intro.tex]: should be changed on the homepage too

Comment (MaxNoe) [reply intro/content/intro.tex]: Yes, but that's a different repo

Comment (chrbeckm) [reply intro/content/intro.tex]: Correct, then I can change it there as well.

Comment (MaxNoe) [reply intro/content/intro.tex]: So, approve?

**Pull request merged**